modules/metering-uninstall.adoc

```diff
@@ -3,25 +3,66 @@
 // * metering/metering-uninstall.adoc
 
 [id="metering-uninstall_{context}"]
-= Uninstalling metering from {product-title}
+= Uninstalling the Metering Operator from {product-title}
 
-You can remove metering from your cluster.
+You can remove the Metering Operator and the associated metering components from your cluster.
+
+[NOTE]
+====
+Metering does not manage or delete any S3 bucket data. If you used Amazon S3 for storage, any S3 buckets used to store metering data must be manually cleaned up.
+====
+
+== Removing the Metering Operator
+
+You can remove the Metering Operator by following the documentation on xref:../operators/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster[deleting Operators from a cluster].
+
+== Uninstalling metering
+
+To uninstall metering, you must remove the MeteringConfig resource and delete the `openshift-metering` namespace.
 
 .Prerequisites
 
-* Metering must be installed.
+* You have uninstalled Metering following the documentation listed in the previous step.
 
 .Procedure
 
-To remove metering:
+. Remove all resources created by the Metering Operator:
++
+----
+$ oc --namespace openshift-metering delete meteringconfigs <1>
+----
++
+<1> The name of the MeteringConfig resource that is present in the openshift-metering namespace.
++
+
+. Once the previous step is complete, verify that all Pods in the openshift-metering namespace are deleted.
```

Comment:
Is there a command to check that all the pods are deleted?

Reply:
Yeah, you could run `$ oc --namespace openshift-metering get pods` and check that the no Pods get returned or all the Pods are reporting a terminating state.